VATSpy.dat

```diff
@@ -18069,16 +18069,18 @@ NFFF|Nadi Oceanic||NFFF
 NFFN|Nadi||NFFN
 NVVV|Port Vila||NVVV
 NSTU|Pago Pago||NZZO
-NTTT|Tahiti Oceanic||NTTT
+NTTT|Tahiti Radio||NTTT
 NWWW|Noumea||NWWW
-NZZO|Auckland Oceanic||NZZO
+NZZO|Auckland Radio||NZZO
+NZZO|Auckland Radio East|NZZO-E|NZZO
 NZAA|Auckland||NZAA
+NZAA|Auckland|NZAA-R|NZAA
 NZCB|Christchurch (Bay)|NZCH-B|NZCB
 NZCT|Christchurch (Taranaki)|NZCH-T|NZCT
 NZCK|Christchurch (Kaikoura)|NZCH-K|NZCK
 NZCS|Christchurch (South)|NZCH-S|NZCS
-NZOH|Ohakea ||NZOH
-NZCM|McMurdo||NZCM
+NZOH|Ohakea||NZOH
+NZCM|McMurdo (Mac)||NZCM

```

Comment:
Out of curiosity, why has (Mac) been included here? As far as I can see, there is only one FIR named McMurdo here, no subsectors like Christchurch

Reply:
The item in parenthesis is typically that station's callsign. For example, Christchurch Control (Taranaki)'s RTF ident is Taranaki/Naki. In this case, NZCM's RTF ident is Mac. Roughly similar to how VATPAC handle theirs.